spec/requests/summary_documents_api_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+
+RSpec.describe 'POST /api/v1/summary_documents' do
+  scenario 'create a valid summary document activity' do
+    given_the_user_is_a_pension_wise_api_user do
```

Comment:
One thing I just noticed - this is only working since that user has the signin permission, which technically means any user can access this API. We should limit this to the `PENSION_WISE_API_PERMISSION` to ensure only our API user can access this [per the example here](https://github.com/guidance-guarantee-programme/telephone_appointment_planner/blob/7f368c94e07b8d8ddb0ee65def2a5ce14f0cbbac/app/controllers/api/v1/appointments_controller.rb#L6)